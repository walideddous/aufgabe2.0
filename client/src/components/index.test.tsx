import React from "react";
import Aufgabe from "./index";
import { shallow, mount } from "enzyme";
import toJSON from "enzyme-to-json";

const setUp = () => {
  const component = shallow(<Aufgabe />);
  return component;
};

describe("Aufgabe component => main component", () => {
  let mountWrapper: any;

  beforeEach(() => {
    mountWrapper = setUp();
  });

  it("Match snapShot", () => {
    expect(toJSON(mountWrapper)).toMatchSnapshot();
  });
});
