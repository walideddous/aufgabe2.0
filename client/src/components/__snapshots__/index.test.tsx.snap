// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aufgabe component => main component Match snapShot 1`] = `
<div
  className="Prototyp"
  style={
    Object {
      "position": "relative",
    }
  }
>
  <Row
    gutter={
      Array [
        8,
        8,
      ]
    }
  >
    <Col
      xs={24}
    >
      <Memo(LoadStopSequence)
        currentStopSequence={Object {}}
        handleDeleteStopSequence={[Function]}
        handleUpdateAfterSave={[Function]}
        onClearAll={[Function]}
        onSendRequest={[Function]}
        ondisplayStopSequence={[Function]}
        stateDND={
          Object {
            "trajekt": Object {
              "items": Array [],
              "title": "Stop sequence",
            },
            "vorschlag": Object {
              "items": Array [],
              "title": "Suggestion",
            },
          }
        }
        stopSequenceList={Array []}
      />
    </Col>
    <Col
      xs={24}
    >
      <Memo(SaveStopsSequenceForm) />
    </Col>
    <Memo(Map)
      currentStopSequence={Object {}}
      distance={Array []}
      handleSelectAutoSearch={[Function]}
      onAddAfterSelected={[Function]}
      onAddBeforSelected={[Function]}
      onDeleteMarkerFromMap={[Function]}
      selectMarkerOnMap={[Function]}
      stateDND={
        Object {
          "trajekt": Object {
            "items": Array [],
            "title": "Stop sequence",
          },
          "vorschlag": Object {
            "items": Array [],
            "title": "Suggestion",
          },
        }
      }
      stations={Array []}
    />
    <Col
      xs={24}
      xxl={12}
    >
      <Memo(DragDrop)
        handleAddStopsOnCLick={[Function]}
        handleDragEnd={[Function]}
        onDelete={[Function]}
        onclick={[Function]}
        stateDND={
          Object {
            "trajekt": Object {
              "items": Array [],
              "title": "Stop sequence",
            },
            "vorschlag": Object {
              "items": Array [],
              "title": "Suggestion",
            },
          }
        }
      />
    </Col>
  </Row>
</div>
`;
